python/ct/client/db/cert_desc_test.py

```diff
@@ -1,21 +0,0 @@
-#!/usr/bin/env python
-import unittest
-from ct.client.db import cert_desc
-from ct.crypto import cert
-CERT = cert.Certificate.from_der_file("ct/crypto/testdata/google_cert.der")
-class CertificateDescriptionTest(unittest.TestCase):
-    def test_from_cert(self):
-        desc = cert_desc.CertificateDescription.from_cert(CERT)
-        self.assertEqual(desc.der, CERT.to_der())
```

Comment:
Like, this line could become something like this:

    self.assertEqual(desc.der_cert, CERT.to_der())